content/en/docs/tasks/security/cert-management/custom-ca-k8s/index.md

```diff
@@ -276,13 +276,24 @@ When the workloads are deployed, they send CSR requests with related signer info
 
 ## Cleanup
 
-* Remove the `istio-system`, `foo` and `bar` namespaces:
-
-    {{< text bash >}}
-    $ kubectl delete ns istio-system
-    $ kubectl delete ns foo
-    $ kubectl delete ns bar
-    {{< /text >}}
+*   Remove the sample applications `sleep` and `httpbin`:                                                                                                                                                          
+                                                                                                                                                                                                                   
+    {{< text bash >}}                                                                                                                                                                                              
+    $ kubectl delete -f samples/sleep/sleep.yaml -n foo                                                                                                                                                            
+    $ kubectl delete -f samples/httpbin/httpbin.yaml -n foo                                                                                                                                                        
+    {{< /text >}}                                                                                                                                                                                                  
+                                                                                                                                                                                                                   
+*   Uninstall Istio from the cluster:                                                                                                                                                                              
+                                                                                                                                                                                                                   
+    {{< text bash >}}                                                                                                                                                                                              
+    $ istioctl uninstall --purge -y                                                                                                                                                                                
```

Comment:
This document doesn't tell the user to install istio using `istioctl install` so it doesn't seem appropriate to include `istioctl uninstall` in the cleanup instructions. That said, the current cleanup instructions are deleting the istio-system ns, even though the doc seems to start with the assumption that Istio is already installed before you start. Bottom line is that this doc needs to be fixed more than just tweaking the cleanup instructions, and especially it needs a test.

@majeshps if you are willing to pick up the bigger issue of fixing the doc and add a test, that would be great. Otherwise, I'm inclined to leave this doc unchanged for now.